bats/fb-content-katello.bats

```diff
@@ -198,8 +199,3 @@ EOF
   skip # because of above
   puppet agent --test && grep -q Lorem /tmp/dummy
 }
-
-@test "Delete an Organization" {
```

Comment:
Any reason to remove this? Or do you think we should use the other file to do this instead?